.github/workflows/build_and_run_test_app_usb.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       fail-fast: false
       matrix:
         idf_ver: ["release-v5.0", "release-v5.1", "release-v5.2", "release-v5.3", "latest"]
-        idf_target: ["esp32s2"]
+        idf_target: ["esp32s2", "esp32p4"]
```

Comment:
In case this fails from idf < 5.3 you will have to add something like

```
exclude:
          - idf_ver: "release-v5.1"
            idf_target: esp32p4 # ESP32-P4 support started with version 5.3
```

Reply:
Good point, thx. Now added to the .yml file.